DSharpPlus/Net/Rest/RateLimitStrategy.cs

```diff
@@ -153,20 +168,30 @@ TState state
         }
     }
 
-    private Outcome<HttpResponseMessage> SynthesizeInternalResponse(string route, DateTime retry, string scope)
+    private Outcome<HttpResponseMessage> SynthesizeInternalResponse(string route, DateTime retry, string scope, Ulid traceId)
     {
         string waitingForRoute = scope == "route" ? " for route hash" : "";
 
+        this.logger.LogTrace
```

Comment:
do we need this?

Reply:
I added it because you didnt want ids in debug statements

Reply:
can we switch on logger.LogLevel then, if available? (or IsEnabled(Trace))? having two entries for the same thing us a bit eh

Reply:
I will take a look at it